MedscribeUI/src/components/Utilities/ThreeBtnSelector.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button } from '@mantine/core';
+import { useState, useEffect } from 'react';
+
+interface ThreeBtnSelectorProps {
+  buttonLabelOptions: string[];
+  initialSelected?: string;
+  onSelect: (value: string) => void;
+}
+
+function ThreeBtnSelector({ buttonLabelOptions, initialSelected = '', onSelect }: ThreeBtnSelectorProps) {
+  const [selected, setSelected] = useState(initialSelected);
+
+  useEffect(() => {
```

Comment:
remove this. we will pass in the state it should render as a prop called selectedPronoun and a handler that updates it state.

Reply:
btn selector is used for more than just pronouns so i used a generic name